etl/steps/data/garden/demography/2023-10-10/riley_2005.countries.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Isn't (Riley) good enough?

Reply:
This has not surfaced in Grapher for the time being, so not really important. I guess (Riley 2005) is slightly more complete.